train/train_classifier_keras.py

```diff
@@ -0,0 +1,77 @@
+import glob
+import tensorflow as tf
+from tensorflow import keras
+import collect_args
+import goog_helper
+
+def _parse_function(example_proto):
+    """
+    Function for converting TFRecordDataset to uncompressed image pixels + labels
+    :return:
+    """
+    feature_description = {
+    'image/class/label': tf.io.FixedLenFeature([], tf.int64, default_value=0),
+    'image/encoded': tf.io.FixedLenFeature([], tf.string, default_value=''),
+    'image/format': tf.io.FixedLenFeature([], tf.string, default_value=''),
+    'image/height': tf.io.FixedLenFeature([], tf.int64, default_value=0),
+    'image/width': tf.io.FixedLenFeature([], tf.int64, default_value=0),
+    }
+
+    # Parse the input `tf.Example` proto using the dictionary above.
+    example = tf.io.parse_single_example(example_proto, feature_description)
+    image = tf.image.decode_image(example['image/encoded'], channels=3)
+
+    #Resizing images in training set because they are apprently rectangular much fo the time
+    if example['image/height'] != 299 or example['image/width'] != 299:
+        image = tf.image.resize(tf.reshape(image, [example['image/height'], example['image/width'], 3]), [299, 299])
+        image = tf.cast(image, tf.uint8)
+
+    image = tf.reshape(image, [299, 299, 3]) #weird workaround because decode image doesnt get shape
+    label = tf.one_hot(example['image/class/label'], depth=2)
+
+    image = (tf.cast(image, tf.float32) - 128) / 128.0
+    return [image, label]
+
+
+def main():
+    reqArgs = [
+        ["i", "inputDir", "local directory containing both smoke and nonSmoke images"],
+        ["o", "outputDir", "local directory to write out TFRecords files"],
+    ]
+    optArgs = [
+        ["t", "trainPercentage", "percentage of data to use for training vs. validation (default 90)"]
+    ]
```

Comment:
The code doesn't use this argument, so you can just remove this and use an empty list.